.github/workflows/hardware-smoke.yml

```diff
@@ -158,108 +158,44 @@ jobs:
             runs-on:
               - p150a-jtag
     runs-on: ${{ matrix.config.runs-on }}
-    env:
-      "ZEPHYR_SDK_INSTALL_DIR": /opt/toolchains
     container:
-      image: ghcr.io/zephyrproject-rtos/ci-repo-cache:v0.27.4.20241026
+      image: ghcr.io/tenstorrent/tt-metal/upstream-tests-bh:v0.58.0-rc26-5-g668476a4d2
       volumes:
         - /dev/hugepages-1G:/dev/hugepages-1G
-        - /opt/tenstorrent/fw/stable:/opt/tenstorrent/fw/stable
-        - /opt/tenstorrent/twister:/opt/tenstorrent/twister
-        - /opt/tenstorrent/bin/openocd-rtt:/opt/tenstorrent/bin/openocd-rtt
+        - /dev/hugepages:/dev/hugepages
       options: '--device /dev/tenstorrent --device /dev/bus/usb --privileged'
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          path: tt-zephyr-platforms
-          ref: ${{ github.ref }}
-      - uses: ./tt-zephyr-platforms/.github/workflows/prepare-zephyr
-        with:
-          app-path: tt-zephyr-platforms
-
-      - name: Generate board names
+    env:
+      ARCH_NAME: blackhole
+    defaults:
+      run:
         shell: bash
+        working-directory: /home/user/tt-metal/
+    steps:
+      - name: Install Unzip
         run: |
-          case "${{ matrix.config.board }}" in
-            p100) SMC_BOARD=tt_blackhole@p100/tt_blackhole/smc;;
-            p100a) SMC_BOARD=tt_blackhole@p100a/tt_blackhole/smc;;
-            p150a) SMC_BOARD=tt_blackhole@p150a/tt_blackhole/smc;;
-            p300a) SMC_BOARD=tt_blackhole@p300a/tt_blackhole/smc;;
-            *) echo "Unknown board: ${{ matrix.config.board }}"; exit 1;;
-          esac
-          case "${{ matrix.config.board }}" in
-            p100) BMC_BOARD=tt_blackhole@p100/tt_blackhole/bmc;;
-            p100a) BMC_BOARD=tt_blackhole@p100a/tt_blackhole/bmc;;
-            p150a) BMC_BOARD=tt_blackhole@p150a/tt_blackhole/bmc;;
-            p300) BMC_BOARD=p300/tt_blackhole/bmc;;
-            *) echo "Unknown board: ${{ matrix.config.board }}"; exit 1;;
-          esac
-          echo "SMC_BOARD=$SMC_BOARD" >> "$GITHUB_ENV"
-          echo "BMC_BOARD=$BMC_BOARD" >> "$GITHUB_ENV"
-
-      - name: run-e2e-tests
```

Comment:
We need to keep these e2e tests running somewhere, and they should run before we execute metal (as a set of basic sanity tests). Personally I think the best option is to use a separate job for metal tests, that depends on the completion of the E2E tests before running